helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -94,4 +103,13 @@ service:
         - resource/add_collector_k8s
         - resourcedetection
       exporters: [signalfx]
+    {{- if .Values.otelK8sClusterReceiver.k8sEventsEnabled }}
+    logs:
+      receivers:
+        - smartagent/kubernetes-events
+      processors:
+        - resource/add_collector_k8s
```

Comment:
`resource/add_collector_k8s` is not needed here. Please add these instead 
```
        - memory_limiter
        - batch
        - resource
```

Reply:
added processors.